### PR TITLE
New application documentation

### PR DESCRIPTION
**Pull request by agenius-mohammed-ali**

### CODE REVIEW

File: R/periscope2.R [hunk 1]
```diff
@@ -72,3 +72,10 @@ NULL
         }
     }
 }
+
+
+.onLoad <- function(libname, pkgname) {
+    if ("package:periscope" %in% search()) {
```
Comment (cb4ds) [inline R/periscope2.R:78]: Wrap in a try/catch and put out a console warning if it cannot be unloaded

File: vignettes/new-application.Rmd [hunk 2]
```diff
@@ -66,8 +66,14 @@ This is the main body area for applications where charts and figures, tables, et
 
 ### New Application Template Generation
 
-* **Empty application** - ready for the user to drop in their code!
-* **Sample application** - demonstrates framework features in a working application
+* User have two main types of application to generate:
```
Comment (cb4ds) [inline vignettes/new-application.Rmd:69]: Reword: 
User can generate two main types of applications

**Review by cb4ds — CHANGES_REQUESTED** [review]

File: R/periscope2.R (hunk 1 above)
Comment (agenius-mohammed-ali) [reply R/periscope2.R:78]: done

File: vignettes/new-application.Rmd (hunk 2 above)
Comment (agenius-mohammed-ali) [reply vignettes/new-application.Rmd]: done

**Review by cb4ds — APPROVED** [review]

**Review by cb4ds — CHANGES_REQUESTED** [review]
needs a version bump!

**Review by cb4ds — APPROVED** [review]

**Pull request merged**